### ISSUE TITLE
Running scripts as MA causes log guard error

### ISSUE THREAD

**Issue by ahelsing** [GENI-NSF/geni-portal#942]
Running scripts as the MA means that the chapi functions when they call the log service pass a user_id of the MA and the logging service guard refuses.
The result is that making someone an operator or enabling/disabling a user, etc is not logged in the log service.

For example:
```
12/17/2013 14:12:47:INFO    :amsoil:[sav1] Called: <modify_project_membership>
12/17/2013 14:12:47:INFO    :chapi:SA: portal-sandbox-admin@gpolab.bbn.com: Invoked modify_project_membership Options {'members_to_add': [{'PROJECT_MEMBER': 'urn:publicid:IDN+ch1.gpolab.bbn.com+user+ahelsing', 'PROJECT_ROLE': 'AUDITOR'}]} Arguments {'project_urn': 'urn:publicid:IDN+ch1.gpolab.bbn.com+project+proj-ph'}
12/17/2013 14:12:48:INFO    :chapi:LOG: portal-sandbox-admin@gpolab.bbn.com: Invoked log_event Options {} Arguments {'attributes': {'PROJECT': '66db4e9a-b5eb-4ae7-b740-2817c597c62c', 'MEMBER': 'd482ed02-601d-4af4-8455-37b6f4807e8f'}, 'message': u'ma Added member Aaron Helsinger in role AUDITOR to project proj-ph', 'user_id': ...
12/17/2013 14:12:48:ERROR   :amsoil:[logv1] [ARGUMENT] ARGUMENT_ERROR (Unknown member uid: 4811d309-bc04-4726-b2e7-408d63848087 )
12/17/2013 14:12:48:INFO    :chapi:LOG: portal-sandbox-admin@gpolab.bbn.com: Result from log_event: {'output': '[ARGUMENT] ARGUMENT_ERROR (Unknown member uid: 4811d309-bc04-4726-b2e7-408d63848087 )', 'code': 3, 'value': None}
12/17/2013 14:12:48:INFO    :chapi:SA: portal-sandbox-admin@gpolab.bbn.com: ma Added member Aaron Helsinger in role AUDITOR to project proj-ph
12/17/2013 14:12:48:INFO    :chapi:SA: portal-sandbox-admin@gpolab.bbn.com: Result from modify_project_membership: {'output': None, 'code': 0, 'value': None}
```

 _Imported from trac ticket [#942](http://trac.gpolab.bbn.com/proto-ch/ticket/942),  created by ahelsing on 12-17-2013 at 20:22, last modified: 01-13-2014 at 13:56_

**Comment by ahelsing** [comment]
Not all scripts apparently. Maybe even only geni-add-project-member?

_Trac comment by ahelsing on 12-17-2013 at 20:43_

**Comment by ahelsing** [comment]
With the fixes to #951 and related, this is no longer true.

What you do see now is an error when you load the project page when it fails to look up the MA's UID:
```
01/13/2014 13:54:12:ERROR   :amsoil:[mav1] [ARGUMENT] ARGUMENT_ERROR (Unknown member uids [['0b2c83d3-369b-4d4e-bb1d-6f5bd1affb4c']] )
```

I'll open a new ticket for that.



_Trac comment by ahelsing on 01-13-2014 at 13:56_
